modules/proxy/mod_proxy_fcgi.c

```diff
@@ -92,9 +95,29 @@ static int proxy_fcgi_canon(request_rec *r, char *url)
         host = apr_pstrcat(r->pool, "[", host, "]", NULL);
     }
 
-    if (apr_table_get(r->notes, "proxy-nocanon")
+    from_handler = apr_table_get(r->notes, "proxy-sethandler") != NULL;
+    if (from_handler
```

Comment:
See https://lists.apache.org/thread/rjh5t17512yxt44cogxwqzscgo4t2m09

Why do we need the `from_handler` variable?

Reply:
This was taken from a larger patch in this thread [1] where there is another use of it.

[1] https://lists.apache.org/thread/vmz22tv2j8bz7vhw98hy3xwvqyp68cdc

Reply:
Axed in b207f2026748fef281b95c6f2e33a958f2eac991